### PR TITLE
Mobile menu

### PR DESCRIPTION
**Pull request by kkmanuu**
**Feature/added-Mobile-Menu**

Added index.html
Added style.css
Added ESLint

### CODE REVIEW

File: script.js [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+  bodyEl.classList.toggle('overflow');
+  // @ts-ignore
+  projectLogo.classList.toggle('opacity');
+  // @ts-ignore
+  navItems.classList.toggle('mobile-nav-list');
+}
+
+// JENERAL OPERATION
+// @ts-ignore
+navBtn.addEventListener('click', operations);
```
Comment (topeogunleye) [inline script.js:20]: Hello,

Thank you for submitting your code for review. I have checked your code and I found that it does not pass the first requirement:

- When the user clicks (or taps) the hamburger button, the mobile menu appears.

I tested your code on different devices and browsers, and I noticed that the mobile menu does not open at all when the hamburger button is clicked. This is a major issue, as it prevents me from testing the other requirements and functionalities of your code.

To fix this issue, you need to check your JavaScript code that is responsible for toggling the mobile menu. You may have some errors or bugs in your code that are preventing the menu from opening. You can use the browser's console or debugger tools to find and fix them. You also need to make sure that you have linked your JavaScript file correctly in your HTML file.

Please kindly make this change and resubmit your code for another review. If you need any help or clarification, feel free to reach out to me.

I appreciate your hard work and dedication. You have a good design and layout for your website. You just need to fix this issue and make sure that your code works as expected.

Sincerely,
Temitope


Please see the video below for more details:
https://www.loom.com/share/cf11646f132b4fb1bc2b086f6badb7bd

After making your mobile menu open when the hamburger icon is clicked, please implement the other requirements which are:

1. When the user clicks (or taps) the close (X) button, the mobile menu disappears.
2. When the user clicks (or taps) any of the mobile menu options, the mobile menu disappears.
3. When the user clicks (or taps) any of the mobile menu options, a correct part of the page is displayed.
4. You must stick to the design as much as possible (e.g., font, colors, images, text, margins) using [the templates in Figma](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=0%3A1).

**Review by topeogunleye — CHANGES_REQUESTED** [review]
Hi @kkmanuu, @mar4444,

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

Try-Again

> ## (Highlights) Good Points: 💪 
> * No linter errors. :heavy_check_mark:
> * Followed Github flow :heavy_check_mark:
> * Professional README file. :heavy_check_mark:
> * PR has a good title and summary :heavy_check_mark:

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- In future pull requests, consider following the recommended branch naming conventions, such as using branch names that reflect the specific feature or task being implemented. This will enhance code organization and collaboration within the team. For example: feature/added-Mobile-Menu.

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag @topeogunleye in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: script.js (hunk 1 above)
Comment (kkmanuu) [reply script.js:20]: @topeogunleye I have added the user clicks (or taps) to the hamburger button so that the mobile menu appears.

**Review by LionRouge1 — APPROVED** [review]
Hi @kkmanuu & @mar4444, 

# STATUS: APPROVED! 🥇 💯 
Your project is complete! There is nothing else to say other than... it's time to merge it :shipit:

done

Good luck! 🎉 


Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**Pull request merged**